repo/packages/C/cerebro/0/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "description": "Cerebro",
+  "framework": false,
+  "maintainer": "https://dcos.io/community",
+  "minDcosReleaseVersion": "1.9",
+  "name": "cerebro",
+  "packagingVersion": "4.0",
+  "postInstallNotes": "",
+  "postUninstallNotes": "DC/OS Cerebro service is being uninstalled.",
+  "preInstallNotes": "Cerebro is a Elasticsearch web administration tool formerly known as Kopf. Cerebro is a standalone application (not an Elasticsearch plugin) complatible with Elasticsearch 5.x and newer.",
+  "selected": true,
```

Comment:
Only white listed packages are allowed to set this to true :)

Reply:
Sorry about that, I've copied that from another framework :)